cmd/spire-server/cli/run/run.go

```diff
@@ -323,6 +342,9 @@ func newDefaultConfig() *serverConfig {
 			Log:            logger,
 			BindAddress:    bindAddress,
 			BindUDSAddress: bindUDSAddress,
+			Experimental: server.ExperimentalConfig{
+				BundleEndpointAddress: &net.TCPAddr{Port: defaultBundleEndpointPort},
```

Comment:
Do we need to set default IP here too? Should probably be 0.0.0.0 ? I wasn't able to observe that as a zero value while playing around with net.TCPAddr

Reply:
This isn't necessary, at least as written. The address is `String()`'d before being passed into the net package and turns into `:<port>`, which is correct.

Reply:
Oh heh, I forgot about that behavior